statchatui/src/Services/fetchData.js

```diff
@@ -0,0 +1,25 @@
+import { fetchStats } from "../Services/api";
+
+export async function fetchData(
```

Comment:
Can you make this more specific? I'm not sure what "data" exactly we're fetching :) I also mentioned this in your previous PR, if you wanted a hint as to what to name it instead.